folium/plugins/locate_control.py

```diff
@@ -0,0 +1,61 @@
+"""Add Locate control to folium Map.
+
+Based on leaflet plugin: https://github.com/domoritz/leaflet-locatecontrol
+"""
+
+from branca.element import CssLink, Figure, JavascriptLink, MacroElement
+
+from jinja2 import Template
+
+
+class LocateControl(MacroElement):
+    """Control plugin to geolocate the user.
```

Comment:
Could you add a line to explain what this plugin does? Something about that it adds a button, and if you click it it will show the current position of your device on the map or something.

Reply:
Hey, I just made the changes as you suggested. I added two extra info's in the docstring that I'm not sure if is the best place to put it. May you check I let me know please?

Reply:
Thanks for adding that info, I think it's clear!